sdk/communication/azure-communication-networktraversal/src/test/java/com/azure/communication/networktraversal/CommunicationRelayClientTestBase.java

```diff
@@ -94,6 +95,7 @@ protected CommunicationIdentityClientBuilder createIdentityClientBuilder(HttpCli
         CommunicationIdentityClientBuilder builder = new CommunicationIdentityClientBuilder();
         builder
             .connectionString(CONNECTION_STRING)
+            .serviceVersion(CommunicationIdentityServiceVersion.V2022_10_01)
```

Comment:
Why do we need to add this? To not break tests while the new API version isn't in Prod yet?

Reply:
Do we want to merge the PR before the API is deployed to PROD or wait? If we wait, then we don't need this change here and elsewhere, right?

Reply:
The reason for adding this service version pointing to previous stable version is because CI pipeline build fails because request points to 2023-10-01 and recordings have version 2022-10-01 for dependent SDK's. Once we release Identity new version, we would update that dependency in dependent SDK's and remove this explicit add of service version (as it will pick latest one)

Reply:
Okay, can you work with the dependent SDK teams to remove this version specifier and update their recordings in a separate PR? (Either they do it or if it's quick you could do it for them)